### ISSUE TITLE
select/copy text in default browser

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#204]
```
What steps will reproduce the problem?
1.long press on text in default browser 
2.celect/copy menu appears but talkback is unable to see this 
3.

What is the expected output? What do you see instead?
as soon as you release long press the select/copy menu disappears. this means 
talkback users are not able to copy text from webpages. 
this needs to be looked at as a priority or implement a completely separate 
select/copy option within the global context talkback menu

What version of the product are you using? On what operating system?
nexus 4, android 4.2.2 latest talkback


Please provide any additional information below.


```

Original issue reported on code.google.com by `redger...@gmail.com` on 13 Mar 2013 at 4:52
- Merged into: #176

**Comment by GoogleCodeExporter** [comment]
Original comment by `caseybur...@google.com` on 29 May 2013 at 7:44
- Changed state: **Duplicate**
